autoscheduler/frontend/src/components/CourseSelectColumn/CourseSelectColumn.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { useSelector, useDispatch } from 'react-redux';
+import { Button } from '@material-ui/core';
+import * as styles from './CourseSelectColumn.css';
+import { RootState } from '../../redux/reducers';
+import { CourseCardArray, CustomizationLevel } from '../../types/CourseCardOptions';
+import CourseSelectCard from './CourseSelectCard/CourseSelectCard';
+import { addCourseCard, removeCourseCard } from '../../redux/actions';
+
+/**
+ * Renders a column of CourseSelectCards, as well as a button to add course cards
+ */
+const CourseSelectColumn: React.FC<RouteComponentProps> = () => {
+  const courseCards = useSelector<RootState, CourseCardArray>(
+    (state) => state.courseCards,
+  );
+
+  const dispatch = useDispatch();
+
+  const rows: JSX.Element[] = [];
+
+  // Add all of the course cards to rows to be displayed
+  for (let i = 0; i < courseCards.numCardsCreated; i++) {
+    if (courseCards[i]) {
+      rows.push(
+        <div
+          className={styles.row}
+          key={`courseSelectCardRow-${i}`}
+        >
+          <CourseSelectCard
+            key={`courseSelectCard-${i}`}
+            id={i}
+            onRemove={(idx: number): void => {
+              dispatch(removeCourseCard(idx));
+            }}
+          />
+        </div>,
+      );
+    }
+  }
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.columnWrapper}>
+        <div className={styles.courseSelectColumn}>
+          {rows}
+        </div>
+      </div>
+      <Button
+        color="primary"
+        size="medium"
+        variant="contained"
+        id={styles.addCourseButton}
+        onClick={(): void => {
+          dispatch(addCourseCard({ customizationLevel: CustomizationLevel.BASIC, sections: [] }));
+        }}
+      >
+        Add Course
+      </Button>
+    </div>
```

Comment:
What div do you want it to be inside of exactly? The width is originally set by `.container` so they should have the same width.

Reply:
I meant the div containing all the cards (so when the scrollbar appears it has the same width as the other cards), but you could just add a top margin to the button so it doesn't touch the scrollbar/course cards and keep the original width, since it might be awkward to try to anchor it to the bottom of that div and I don't think we necessarily have to match the widths of the course cards and add course buttons.

Reply:
Was this fixed by #171?

Reply:
Yes, this looks how I wanted it to now